examples/platform/telink/common/src/AppTaskCommon.cpp

```diff
@@ -103,6 +104,14 @@ Identify sIdentify = {
 uint8_t sTestEventTriggerEnableKey[TestEventTriggerDelegate::kEnableKeyLength] = { 0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77,
                                                                                    0x88, 0x99, 0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff };
 #endif
+
+class AppCallbacks : public AppDelegate
+{
+public:
+    void OnCommissioningWindowClosed() override { chip::DeviceLayer::Internal::BLEMgr().Shutdown(); }
```

Comment:
Hmm. What if someone tries to open the commissioning window again e.g. by pressing a button?

Reply:
@Damian-Nordic 
To open the commissioning window the method **CommissioningWindowManager::StartAdvertisement()** is used.

This method calls the method **chip::DeviceLayer::ConnectivityMgr().SetBLEAdvertisingEnabled(true);** , which is redefined by vendor.

For Telink platforms enabling the advertisement will enable the BLE interface automatically.

![pr_26786](https://github.com/project-chip/connectedhomeip/assets/36621698/3ab2e21d-b2bf-49d0-a598-66faa6afa610)

So during commissioning window closing the BLE interface will be shutting down and the flag **mBLERadioInitialized** will be set to false. 
During opening the commissioning window the BLE Manager will check this variable and enable interface if the variable is false.